components/Card.js

```diff
@@ -2,24 +2,38 @@ import Image from "next/image";
 import styled from "styled-components";
 import { Button } from "./Button";
 import Delete from "./icons/Delete";
+import { useRouter } from "next/router";
+
+export default function Card({
+  date,
+  id,
+  weight,
+  handleDelete,
+  height,
+  feastTime,
+}) {
+  const { pathname } = useRouter();
 
-export default function Card({ day, handleDelete }) {
   return (
     <CardElement>
-      <DeleteButton
-        type="button"
-        aria-label="delete data"
-        onClick={() => handleDelete(day.id)}
-      >
-        <Delete height={20} width={20} alt="delete" />
-      </DeleteButton>
-      <p>Date: {day.date}</p>
-      <p>Weight: {day.weight}</p>
-      <p>Height: {day.height}</p>
-      <p>
-        Nurse time: {day.feastTime.substr(0, 2)}:{day.feastTime.substr(2, 2)}:
-        {day.feastTime.substr(4, 2)}
-      </p>
+      {pathname === "/" && (
+        <DeleteButton
+          type="button"
+          aria-label="delete data"
+          onClick={() => handleDelete(id)}
+        >
+          <Delete height={20} width={20} alt="delete" />
```

Comment:
why aren't you setting height and width within the styled component?

Reply:
Good question, long answer :D
These aren't styled components, but svgs, which I automatically coverted to components, which I then import.
I do that, so that I may switch color & size inline, instead of having to change that in each icon :)